src/services/paras/ParasService.ts

```diff
@@ -223,18 +223,45 @@ export class ParasService extends AbstractService {
 			phase: IOption<AuctionPhase>,
 			winning;
 		if (auctionInfoOpt.isSome) {
+			/**
+			 * AuctionInfo:::<T>:get() when Some, it returns a tuple where the first item is the
```

Comment:
If `AuctionInfo:::<T>:get()` is Some  it returns a tuple where the first item is the